### PR TITLE
chore: added blog -Resolving merge conflict on Github in category,subcategory

### PR DESCRIPTION
**Pull request by devasyarajguru**
## Changes proposed
Added a blog link to Opensource section with proper Name , description, url , category , subcategory

## Screenshots

Resolving guide

## Note to reviewers

None

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Great job @devasyarajguru! 😄 LGTM! 😄

**Comment by devasyarajguru** [comment]
I have to fix the changes you have told above? I have to fix it again in my local setup(VS Code) and again add, commit, push changes ?? That's it? @CBID2

**Comment by devasyarajguru** [comment]
> > I have to fix the changes you have told above? I have to fix it again in my local setup(VS Code) and again add, commit, push changes ?? That's it? @CBID2
> 
> Have you checked the Contributing file before making this PR @devasyarajguru?

Yes I have checked it. Why?

**Comment by devasyarajguru** [comment]
Hey @CBID2 I just misunderstand the comment. Actually I thought I have to fix the change but it's already changed by you. I hope it is fine?

**Comment by CBID2** [comment]
> Hey @CBID2 I just misunderstand the comment. Actually I thought I have to fix the change but it's already changed by you. I hope it is fine?

It's fine @devasyarajguru! :) Rupali just needs to approve your pull request.

**Comment by devasyarajguru** [comment]
> > Hey @CBID2 I just misunderstand the comment. Actually I thought I have to fix the change but it's already changed by you. I hope it is fine?
> 
> It's fine @devasyarajguru! :) Rupali just needs to approve your pull request.

Thank you very much @CBID2 @rupali-codes . You both are very supportive and active. It was great to contribute! Looking forward to do more!

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @devasyarajguru! :) There are some branch conflicts that need to be solved.

**Comment by devasyarajguru** [comment]
For solving the conflicting branch , do I have to perform [git pull upstream main] ?? and then git push?

**Comment by devasyarajguru** [comment]
Please checked this. I have resolved the issues @CBID2

**Review by CBID2 — DISMISSED** [review]
Great job @devasyarajguru!😄 LGTM! 😄

**Comment by devasyarajguru** [comment]
> Great job @devasyarajguru!😄 LGTM! 😄

Thanks @CBID2 . When will be my PR merged?

**Comment by CBID2** [comment]
> > Great job @devasyarajguru!😄 LGTM! 😄
> 
> Thanks @CBID2 . When will be my PR merged?

When you get an approved review by @rupali-codes @devasyarajguru! 😊

**Comment by devasyarajguru** [comment]
> > > Great job @devasyarajguru!😄 LGTM! 😄
> > 
> > 
> > Thanks @CBID2 . When will be my PR merged?
> 
> When you get an approved review by @rupali-codes @devasyarajguru! 😊

Okay no problem. But just tell me that I have resolved conflicts properly or not?

**Review by rupali-codes — COMMENTED** [review]
This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀

**Comment by devasyarajguru** [comment]
> This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀

Does that count my contribution? @CBID2 @rupali-codes , and will this PR will remain open? Please explain it as I am a beginner.

**Comment by devasyarajguru** [comment]
> > This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀
> 
> Does that count my contribution? @CBID2 @rupali-codes , and will this PR will remain open? Please explain it as I am a beginner.

Also I have done valid contribution should'nt my PR gets merged??

**Comment by rupali-codes** [comment]
> > > This site was already added in [open-source-tools](https://www.linkshub.dev/open-source-tools) 👀
> > 
> > 
> > Does that count my contribution? @CBID2 @rupali-codes , and will this PR will remain open? Please explain it as I am a beginner.
> 
> Also I have done valid contribution should'nt my PR gets merged??

Yes your PR is completely valid, but the resource has already been added. Could you please work on some other issues and close this PR? I really appreciate your time and commitment, but we can not have duplicate resources for the same thing.

**Comment by devasyarajguru** [comment]
Subject: Disappointment Regarding Unmerged Pull Request

Dear @CBID2 @rupali-codes ,

I hope this message finds you well. Firstly, I want to express my sincere appreciation for the opportunity to contribute to the project and for the efforts you all put into maintaining it. I understand that you receive numerous pull requests on a regular basis, making the task of managing and merging them quite challenging.

I am writing to you today with a sense of disappointment regarding the unmerged pull request (PR) I recently submitted. I had worked diligently on addressing an issue that was assigned to me, and I firmly believed that my proposed solution was both valid and beneficial to the project.

I would like to remind you that the project follows the "First Come First Serve" principle when it comes to assigning and resolving issues. In accordance with this principle, I had made a considerable effort to understand the problem thoroughly, brainstorm a solution, and implement it in a way that adhered to the project's guidelines and best practices. I dedicated significant time and energy to ensure the quality and correctness of the code in my PR.

To my surprise, upon submitting my PR, I discovered that a similar solution had already been merged into the project. I understand that situations like this can arise due to various factors, including oversight or lack of visibility on my part. Nevertheless, I genuinely believed that my contribution would bring value to the project and that it deserved careful consideration.

Once again, I want to express my gratitude for the opportunity to contribute to this project and for your ongoing commitment to its maintenance. I sincerely hope that we can engage in a constructive dialogue regarding my concerns, as I am eager to learn and grow as a member of this community.

Thank you for your time and consideration.

Yours sincerely,

Devasya

**Comment by rupali-codes** [comment]
If you don't want to close the PR unmerged, kindly refere replacing the resources with something else.
You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

**Comment by devasyarajguru** [comment]
> If you don't want to close the PR unmerged, kindly refer replacing the resources with something else. You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

Ohkay so can I replace this resource in this PR only?

**Comment by rupali-codes** [comment]
> > If you don't want to close the PR unmerged, kindly refer replacing the resources with something else. You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> 
> Ohkay so can I replace this resource in this PR only?

yeah

**Comment by devasyarajguru** [comment]
> > > If you don't want to close the PR unmerged, kindly refer replacing the resources with something else. You may add [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> > 
> > 
> > Ohkay so can I replace this resource in this PR only?
> 
> yeah, can also do in the issue, so it won't create confusions

Updated the website link as mentioned by @rupali-codes . Also updated issue which closes #674. Title and Description updated of PR #749

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hey, just remove the empty line, other than that, looks great to me

File: database/opensource/blogs.json [hunk 1]
```diff
@@ -22,9 +22,9 @@
       "subcategory": "opensource_blogs"
    },
    {
-      "name": "Open Source Guides",
-      "description": "This blog will help you learn about everything Open Source",
-      "url": "https://opensource.guide/",
+      "name": "Resolving merge conflict on Github",
+      "description": "This blog will help you to resolve merge conflict on Github",
```
Comment (CBID2) [inline database/opensource/blogs.json:25]: [Proofreading]
```suggestion
      "description": "This blog will help you to resolve merge conflicts on GitHub",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @devasyarajguru! :) I pointed out some proofreading errors that could be improved.

**Comment by rupali-codes** [comment]
> Hi @devasyarajguru! :) I pointed out some proofreading errors that could be improved.

let's merge it, 😄

**Comment by rupali-codes** [comment]
@CBID2 could you approve the PR please :)

**Review by rupali-codes — APPROVED** [review]

**Comment by devasyarajguru** [comment]
> Hi @devasyarajguru! :) I pointed out some proofreading errors that could be improved.

What are proofreading errors? I don't know , I am contributing first time. And No problem , it would be great for me to improve this particular thing and then afterwards you can merge my PR. It would be great for me if you help me pointing out my improvements! Thanks! @rupali-codes @CBID2

**Review by CBID2 — APPROVED** [review]
LGTM!

**Comment by CBID2** [comment]
> Hey @CBID2 and @rupali-codes . Thanks for co-operating and supporting me. I want to know is there any improvement to do in this PR? Also @CBID2 I want to make sure that I will improve upon the last comment you gave me about proofreading errors. I want to know is there anything from my side on this now?

You’re welcome @devasyarajguru! We just need @rupali-codes to force push your PR. As far as improving on this pull request, not necessary

**Comment by devasyarajguru** [comment]
Thank You very much @rupali-codes and @CBID2 . You guys are very supportive and active. It was great collaborating with you!😊

**Comment by rupali-codes** [comment]
> Thank You very much @rupali-codes and @CBID2 . You guys are very supportive and active. It was great collaborating with you!😊

Its great to have you :)
Thanks

**Pull request merged**